db/models.js

```diff
@@ -28,7 +29,31 @@ function fetchArticlesById(articleId) {
     WHERE articles.article_id = $1`;
     return db.query(sqlQuery, [id]).then((response) => {
         return response;
+    }).catch((error) => {
+        console.error("error in fetchArticlesById", error);
+        return Promise.reject({status: 400, error: "Invalid query"});
```

Comment:
Please remove console logs from finalised code 🙏 
You only need one catch block throughout the promise chain, so there's no need for one in the model, you can just have one in the controller, so you can remove this one.
The promise shouldn't be rejected in the catch block, rejecting the promise causes an error which is caught in the catch block, have a look at the error catching notes for examples of this https://notes.northcoders.com/courses/js-back-end/error-handling#3-testing-and-handling-errors